src/dfx/src/lib/extension/manifest/extension.rs

```diff
@@ -60,6 +60,7 @@ pub struct ExtensionSubcommandArgOpts {
     pub about: Option<String>,
     pub long: Option<String>,
     pub short: Option<char>,
+    pub multiple: Option<bool>,
```

Comment:
beyond the scope of this PR, but this should have doc comments, and we should generate a JSON schema.

Also, especially with bool arguments that default to false, we can simplify things a little bit by using `bool` (default false) rather than `Option<bool>` (default None, assumed to be false):
```suggestion
    #[serde(default)]
    pub multiple: bool,
```

Reply:
thanks! 
https://dfinity.atlassian.net/browse/SDK-1180

I'll change it now from `Option<bool>` to `bool`